pkg/dumper/postgres/dumper.go

```diff
@@ -70,6 +71,7 @@ func (p *pgDumper) dumpTables(done chan<- struct{}, configTables config.Tables)
 	var wg sync.WaitGroup
 	wg.Add(len(tables))
 	for _, tbl := range tables {
+		log.WithField("table", tbl).Info("Dumping table data...")
```

Comment:
Why not `logger := log.WithField("table", tbl)` and use that here and on line 79, 94, 98?